main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
```

Comment:
don't forget to import functions from `utils.go` here

Reply:
Go make functions visible from different files of the same package

Reply:
`go run *.go`